### PR TITLE
MGMT-1679 Call coreos-installer without image-url

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by nmagnezi** [comment]
Will submit another iteration for this PR soon.

**Comment by eranco74** [comment]
@tsorya notice we are dropping the rhcos image from the mapping

**Comment by nmagnezi** [comment]
/hold
testing

**Comment by nmagnezi** [comment]
Looks good. The cluster installed successfully.
A snipped from one of the nodes
```golang
Oct 01 13:54:53 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:54:53Z" level=info msg="Writing image and ignition to disk"
Oct 01 13:54:53 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:54:53Z" level=info msg="Installing Red Hat Enterprise Linux CoreOS 46.82.202009091306-0 (Ootpa) x86_64 (512-byte sectors)\n"
Oct 01 13:54:54 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:54:54Z" level=info msg="Read disk 100.9 MiB/3.5 GiB (2%)\n"
Oct 01 13:54:55 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:54:55Z" level=info msg="Read disk 218.5 MiB/3.5 GiB (6%)\n"
...
<snipped>
...
Oct 01 13:56:07 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:56:07Z" level=info msg="Read disk 3.5 GiB/3.5 GiB (100%)\n"
Oct 01 13:56:28 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:56:28Z" level=info msg="Writing Ignition config\n"
Oct 01 13:56:28 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:56:28Z" level=info msg="Install complete.\n"
Oct 01 13:56:28 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:56:28Z" level=info msg="Done writing image to disk"
Oct 01 13:56:28 test-infra-cluster-assisted-installer-master-1 installer[2832]: time="2020-10-01T13:56:28Z" level=info msg="Updating node installation stage: Rebooting - "
```

**Comment by eranco74** [comment]
https://github.com/openshift/assisted-installer/pull/16#issuecomment-701965528

File: src/installer/installer.go [hunk 1]
```diff
@@ -111,12 +111,8 @@ func (i *installer) InstallNode() error {
 
 	i.UpdateHostInstallProgress(models.HostStageWritingImageToDisk, "")
 
-	image, _ := utils.GetRhcosImageByOpenshiftVersion(i.OpenshiftVersion)
```
Comment (rollandf) [inline src/installer/installer.go:114]: 'GetRhcosImageByOpenshiftVersion' is not used anymore.
Should we delete it?

Comment (nmagnezi) [reply src/installer/installer.go:114]: Yup.

**Review by eranco74 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**